### ISSUE TITLE
ShuffleNet with latest Keras & TensorFlow 2.0

### ISSUE THREAD

**Issue by dcosmin-97** [opconty/keras-shufflenetV2#6]
Hello ! 

I am having a lot of errors when i try to make this compatible with the latest Keras & Tensorflow 2.0 .
I have to make the training on Kaggle and then to save the model.h5 and use it on Raspberry Pi 4. 

Can you edit the code to be able to run with Keras & Tensorflow latest updates? 

Here you can find the link to Kaggle: https://www.kaggle.com/dcosmin/shufflenet-with-keras.

Thank you !
I really need this ShuffleNet Model for a project !

**Comment by dcosmin-97** [comment]
The first error that was fixed:
Initial : from keras.applications.imagenet_utils import _obtain_input_shape
After:   from keras_applications.imagenet_utils import _obtain_input_shape

Now the problem is with utils library . The block module cannot be recognized.

**Comment by opconty** [comment]
HI,
The code runs on tensorflow 1+,  and it's been a long time without maintaining this repo.
Yes, of course, I'll update the code when I have time.
Thanks for your attention.
Best.

**Comment by dcosmin-97** [comment]
Hello ! Can you check your email please? I have send you some more detailed things.

**Comment by opconty** [comment]
I've tested, it works fine for me ,and this is my requirements version:
Keras==2.3.1
tensorflow_gpu==2.1.0
numpy==1.18.2
opencv_python_headless==4.1.2.30
tensorflow==2.2.0
